src/Hal/Report/Cli/Reporter.php

```diff
@@ -4,19 +4,23 @@
 use Hal\Application\Config\Config;
 use Hal\Component\Output\Output;
 use Hal\Metric\Consolidated;
+use Hal\Metric\Metric;
 use Hal\Metric\Metrics;
-
-class Reporter
+use Hal\Report\ReporterInterface;
+use function array_slice;
```

Comment:
Please do not use this syntaxe (incompatible with PHP 5.5)

Reply:
It's weird that tests did not catch this btw... we must look at this..